src/app/modules/side-menu/components/side-menu.component.ts

```diff
@@ -132,18 +126,10 @@ export class SideMenuComponent implements OnInit, OnDestroy {
         .get(['MENU.ERROR_REPORT_DESCRIPTION', 'MENU.CONTACT_REGOBS_ERROR'])
         .toPromise();
       const appVersion = await this.appVersionService.getAppVersion();
-      const email: EmailComposerOptions = {
-        to: settings.errorEmailAddress,
-        subject:
-          `${translations['MENU.CONTACT_REGOBS_ERROR']}: ${
-            this.platform.is('ios') ? 'ios' : ''
-          }` +
+      const subject = `${translations['MENU.CONTACT_REGOBS_ERROR']}: ${this.platform.is('ios') ? 'ios' : ''}` +
           `${this.platform.is('android') ? 'android' : ''}` +
```

Comment:
Hva med å bruke denne i stedet for å hente platform-navnet?
Capacitor.getPlatform()

Du kan også bruke Capacitor.isNativePlatform() i stedet for isAndroidOrIos(this.platform), så slipper du å injecte platform.